conf/airframes/examples/bebop2_indi.xml

```diff
@@ -140,7 +146,7 @@
     <define name="G1_R" value="0.0022"/>
 
     <!-- For the bebop2 we need to filter the roll rate due to the dampers -->
-    <define name="FILTER_ROLL_RATE" value="TRUE"/>
+    <!-- FILTER_ROLL_RATE moved to firmware targets -->
```

Comment:
Why has this been moved to nps target only? Not sure if that's a good idea. Why does this have to be turned off for Gazebo?

Reply:
Setting FILTER_ROLL_RATE to TRUE causes oscillations around the roll axis. I thought this was a gazebo problem, but I've just found out this also happens in jsbsim so it might actually be a problem with this filter instead.

Could you please try to replicate this as follows:
1. Start a flight with the examples/bebop2_indi airframe and rotorcraft_basic flight plan (from master). The flight will be simulated using jsbsim, not gazebo.
2. Follow the flight plan until the standby block.
3. Check the PFD in GCS, does the drone oscillate around the roll axis?
4. Disable the ROLL_FILTER, does the drone remain stable now?

Reply:
@EwoudSmeur I guess you implemented the jbsim indi stuff, can you check this?

Reply:
Discussed with @EwoudSmeur , enabling the roll filter could indeed cause these oscillations. Use of the filter on the real bebop2 is also unnecessary when the dampers are locked (which I've been told is usually the case). How about setting the roll rate filter to off by default, with a comment hinting at its use when flying with unlocked dampers (see latest commit)?